File: examples and misc/Cpp/classes.cpp
#include <iostream>
using namespace std;

// Define a class
class Animal {
public:
    string name;
    int age;

    // Constructor
    Animal(string n, int a) : name(n), age(a) {}

    // Method
    void speak() {
        cout << name << " says hello!" << endl;
    }
};

int main() {
    // Create an instance of Animal
    Animal dog("Buddy", 3);
    dog.speak();
    return 0;
}

File: examples and misc/Python/classes.py
# Define a class
class Animal:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    # Method
    def speak(self):
        print(f"{self.name} says hello!")

# Create an instance of Animal
dog = Animal("Buddy", 3)
dog.speak()
